sormas-ui/src/main/java/de/symeda/sormas/ui/person/PersonEditForm.java

```diff
@@ -403,6 +408,18 @@ protected void addFields() {
 				false,
 				I18nProperties.getValidationError(Validations.afterDate, burialDate.getCaption(), deathDate.getCaption())));
 
+		phoneNumber.addValidator(
+			new PhoneNumberValidator(
+				I18nProperties.getValidationError(Validations.validPhoneNumber)));
+
+		phoneOwner.addValidator(
```

Comment:
The "Phone Owner" is not a phone number. It's the owner of the phone (used e.g. in African countries where there are remote communities with only one or a few phones that a specific person is responsible for).